### DISCUSSION TITLE
ww3 6.07 ww3_grid.inp output boundaries

### DISCUSSION THREAD

**Discussion by JMSnyder1980** [NOAA-EMC/WW3#952] [General]
I am trying to set up output boundaries but when I set them up as specified by the manual I get an error message:

  Output boundary points : 
 --------------------------------------------------

 *** WAVEWATCH III ERROR IN W3GRID : 
     ERROR IN READING FROM INPUT FILE
     IOSTAT =    0

I am attaching my ww3_grid.inp.  Can you tell me how to tweak the output boundary specifications to get the desired results?  Do I need ww3_bound for this?  If so how to I set up the input file for this?

[WW3_grid.inp.txt](https://github.com/NOAA-EMC/WW3/files/11022002/WW3_grid.inp.txt)

**Comment by aliabdolali** [comment]
You need to specify the index of open boundaries in ww3_grid.inp(nml). see ww3_tp2.17 input as an example.
Then you need to run ww3_bounc or ww3_bound. But note ww3_bound(c) need directional spectral density files. see ww3_tp2.17 input as an example.

**Reply by JMSnyder1980** [reply]
I am not quite surer how to specify the index of open boundaries even when looking at ww3_tp2.17.  I thought I specified open boundaries in thee ww3_grid.inp I send you.   It is very fuzzy even with the example what to do.   Also how to I tweak the nml file to match the boundaries I chose for the ww3_grid input file?  Additionally will running ww3_grid take into account what is mentioned in ww3_grid.nml?   However the main issue I need to address is what do you mean by index of open boundaries and where exactly in the ww3_grid.inp and or the ww3_grid.nml file do I specify this?  Is there a tutorial on how to do this?   I really need a walkthrough on this.
